x/stakeibc/keeper/hooks.go

```diff
@@ -35,7 +35,9 @@ func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochIdentifier string, epochN
 					continue
 				}
 				delegateAddress := delegateAccount.Address
-				timeoutHeight := clienttypes.NewHeight(0, 500)
+				// TODO(TEST-89): Set NewHeight relative to the most recent known gaia height (based on the LC)
+				// TODO(TEST-90): why do we have two gaia LCs?
+				timeoutHeight := clienttypes.NewHeight(0, 10000)
```

Comment:
higher is better here to be sure to avoid issues while testing

Reply:
Pushed to 1000000